src/werkzeug/datastructures.py

```diff
@@ -2090,6 +2090,9 @@ class ResponseCacheControl(_CacheControl):
     you plan to subclass it and add your own items have a look at the sourcecode
     for that class.
 
+    .. versionchanged:: 2.1.1
+        ``max_age`` works correctly with an argument of type int.
```

Comment:
This one should be `s_maxage`?  `max_age` is currently ok

Reply:
Good catch! I will correct this nonsense.

Reply:
Done now!